### PR TITLE
PIL-1057:Top of the screen is cut off

### PR DESCRIPTION
**Pull request by divyaBacancy**
Since the value of "top" used with spacing varies for different devices, hence it is behaving differently with different screens. It is better to put the constant value here as done in other screens. I have tested this value in both platform and various devices. It is working fine.

### CODE REVIEW

**Comment by poocart** [comment]
Just a question – have you found out why 60? Is this related to header height size or bottom nav bar side or what?

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js [hunk 1]
```diff
@@ -63,7 +61,7 @@ function KeyBasedAssetTransferIntro() {
         backgroundColor={darkTheme.colors.basic090}
       >
         <Content>
-          <Spacing h={top} />
+          <Spacing h={60} />
```
Comment (mdjastrzebski) [inline src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:64]: If I understand the issue correctly the problem is that top of the screen has a different shade of black there. 
Simple solution would be to change `darkTheme.colors.basic090` to `darkTheme.colors.background`.

Proper solution would be to create a new floating back button component that will allow content to flow underneath itself.

Proposed solution of fixed top spacing does not seem to address that.

**Review by mdjastrzebski — CHANGES_REQUESTED** [review]

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js (hunk 1 above)
Comment (divyaBacancy) [reply src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js]: @mdjastrzebski @IAmKio @poocart Done.

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js [hunk 2]
```diff
@@ -57,12 +58,12 @@ function KeyBasedAssetTransferIntro() {
...
-        headerProps={{ floating: true }}
-        backgroundColor={darkTheme.colors.basic090}
-      >
-        <Content>
+      <Container>
+        <HeaderBlock
+          navigation={navigation}
+          noPaddingTop
+        />
+        <Content showsVerticalScrollIndicator={false}>
```
Comment (mdjastrzebski) [inline src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:66]: 1. If using `Container` from `components/modern/Layout` you probably would want to use `Content` component from there as well as it handles bottom safe are for iPhone X correctly.

2. Not sure if we would want to hide scroll indicator there. 

3. Additionally in the current state the top safe area is applied to the scroll view, making the top padding of content weird. Using `Content` from `modern/Layout` hopefully should solve it. If not then please do some style override on for that.

image

**Review by mdjastrzebski — CHANGES_REQUESTED** [review]
Suggested some small tweaks as I've noticed some layout issues related to safe area handling

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js (hunk 2 above)
Comment (divyaBacancy) [reply src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:64]: @mdjastrzebski Done. Point 2-  scroll indicator was not there previously, so implemented in such a way.

Comment (mdjastrzebski) [reply src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:64]: IMO I would keep the scroll indicator in default mode as there seems to be no UI or UX reason to hide it.

**Review by mdjastrzebski — CHANGES_REQUESTED** [review]
Almost there 👍🏻 

Last small thing would be to set background color of the screen as in design https://app.zeplin.io/project/5fa95ca539e6804a10dec0a4/screen/5fb334b092ad8e77a0404aad

Design says its #000000 (dark.basic090) and the current background color is more like dark grey. This normally would not be an issue, but it makes the image stand out:

image

Please apply the color to both `Content` & `HeaderBlock`.

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js (hunk 2 above)
Comment (mdjastrzebski) [reply src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:64]: @IAmKio are you aware of any reasons why might want to hide scroll indicator on this screen?

**Comment by mdjastrzebski** [comment]
Also noticed that in the designs working on the button is "Upgrade" vs the current implementation "Migrate"

**Comment by divyaBacancy** [comment]
@mdjastrzebski Done. Changed the background color of the screen as in design.

**Review by mdjastrzebski — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**